tests/Repositories/TimeEntryRepositoryTest.php

```diff
@@ -90,6 +90,27 @@ public function test_can_get_specific_active_task_of_logged_in_user_for_given_pr
         $this->assertContains($task2->id, $result->keys());
     }
 
+    /** @test */
+    public function test_can_get_active_task_of_logged_in_user_having_manage_projects_permission_for_given_project()
+    {
+        $monika = factory(User::class)->create();
+
+        $task1 = factory(Task::class)->create(['status' => Task::STATUS_ACTIVE]);
+        $task1->taskAssignee()->attach($monika->id);
+
+        $task2 = factory(Task::class)->create(['status' => Task::STATUS_ACTIVE, 'project_id' => $task1->project_id]);
+        $task2->taskAssignee()->attach($this->defaultUserId);
+        $completedTask = factory(Task::class)->create(['status' => Task::STATUS_COMPLETED]); // this should not return
+        $completedTask->taskAssignee()->attach($this->defaultUserId);
+
+        $this->actingAs($monika);
```

Comment:
don't needed this both lines because default user already has `manage_projects` permission